PhysicsTools/PatAlgos/python/producersLayer1/ootPhotonProducer_cff.py

```diff
@@ -49,3 +49,44 @@
     )
 
 makePatOOTPhotons = cms.Sequence(makePatOOTPhotonsTask)
+
+## For legacy reprocessing
+## Standard Sequences
+from RecoJets.Configuration.CaloTowersES_cfi import *
+from RecoLocalCalo.EcalRecAlgos.EcalSeverityLevelESProducer_cfi import *
+from RecoParticleFlow.PFClusterProducer.particleFlowRecHitPS_cfi import *
+from RecoParticleFlow.PFClusterProducer.particleFlowClusterPS_cfi import *
+## OOT Sequences
+from RecoParticleFlow.PFClusterProducer.particleFlowRecHitOOTECAL_cff import *
+from RecoParticleFlow.PFClusterProducer.particleFlowClusterOOTECALUncorrected_cff import *
+from RecoParticleFlow.PFClusterProducer.particleFlowClusterOOTECAL_cff import *
+from RecoEcal.EgammaClusterProducers.particleFlowSuperClusterOOTECAL_cff import *
+from RecoEgamma.EgammaPhotonProducers.ootPhotonCore_cff import *
+from RecoEgamma.EgammaPhotonProducers.ootPhotons_cff import *
+from RecoEgamma.EgammaIsolationAlgos.pfClusterIsolation_cfi import *
+
+_makePatOOTPhotonsTask = makePatOOTPhotonsTask.copy()
+_makePatOOTPhotonsTask.add(
+    CaloTowerConstituentsMapBuilder,
+    essourceEcalSev,
+    ecalSeverityLevel,
+    particleFlowRecHitPS,
+    particleFlowClusterPS,
+    particleFlowRecHitOOTECAL,
+    particleFlowClusterOOTECALUncorrected,
+    particleFlowClusterOOTECAL,
+    particleFlowSuperClusterOOTECAL,
+    ootPhotonCore,
+    ootPhotons,
+    ootPhotonEcalPFClusterIsolationProducer
```

Comment:
@slava77 

I think I made the changes as you suggested.  However, there are a few modules that are needed in this legacy task that already included in the standard RECO: 

- CaloTowerConstituentsMapBuilder
- essourceEcalSev 
- ecalSeverityLevel
- particleFlowRecHitPS
- particleFlowClusterPS
- ootPhotonEcalPFClusterIsolationProducer

I added these (and there imports) to this file.  Please review the changes to this file and RecoEgamma/EgammaPhotonProducers/python/ootPhotonSequence_cff.py and let me know if it needs more modification.